practitioner-service/src/main/java/uk/ac/ox/ndph/mts/practitioner_service/validation/RoleAssignmentValidation.java

```diff
@@ -35,12 +39,13 @@ public ValidationResponse validate(RoleAssignment entity) {
             return new ValidationResponse(false, "roleId must have a value");
         }
 
-        if (!this.roleServiceClient.entityIdExists(entity.getRoleId(), roleServiceClient.noAuth())) {
+        if (!this.roleServiceClient.entityIdExists(entity.getRoleId(), RoleServiceClient.noAuth())) {
```

Comment:
not sure about this..seems odd that we call a class and pass in that class as a param.... couldnt we pass a flag to state whether to bypass auth or check if the headers are null in RoleServiceClient and if so use noAuth?

Reply:
The thing here is that is only a temp solution, @LizaShak is already working on the role-service auth and will be replaced soon with actual authHeaders. WDYT?

Reply:
fair enough if its temp - can you add a comment?

Reply:
Done